### PR TITLE
ROC-4043 claimant provide defendant phone

### PR DESCRIPTION
**Pull request by karthikkannankainos**
### JIRA link
https://tools.hmcts.net/jira/browse/ROC-4043

### Change description
Claimant provides the defendant phone number.

### CODE REVIEW

File: src/main/app/claims/claimModelConverter.ts [hunk 1]
```diff
@@ -246,4 +251,11 @@ export class ClaimModelConverter {
       date_created: payment.date_created
     }
   }
+
+  private static convertPhoneNumber(phone: MobilePhone): string {
+    if (phone === undefined) {
+      return undefined;
+    }
+    return StringUtils.trimToUndefined(phone.number)
+  }
```
Comment (andrewwa-kainos) [inline src/main/app/claims/claimModelConverter.ts]: Doesn't cover `null`, which is also falsy. How about just
`return phone ? StringUtils.trimToUndefined(phone.number) : phone`
then
`undefined` => `undefined`
`null` => `null`
`{"number":"something"}` => `"something"`
`{}` => `undefined`
`{"number":"     "}` => `undefined`

File: src/main/features/claim/views/check-and-send.njk [hunk 2]
```diff
@@ -42,6 +42,9 @@
     }}
 
     {{ row('Email', draftClaim.defendant.email.address, paths.defendantEmailPage.uri, bold = true) }}
+    {% if draftClaim.defendant.mobilePhone %}
+      {{ row('Contact number (optional)', draftClaim.defendant.mobilePhone.number, paths.defendantPhonePage.uri, bold = true) }}
+    {% endif %}
```
Comment (andrewwa-kainos) [inline src/main/features/claim/views/check-and-send.njk]: It should still be shown in the check-and-send page if it's absent, so the user sees that it hasn't been entered and gets the link to go and change it.

File: src/main/app/claims/claimModelConverter.ts (hunk 1 above)
Comment (karthikkannankainos) [reply src/main/app/claims/claimModelConverter.ts]: Done. Unable to send phone back as type clash. so sending undefined back.

File: src/main/features/claim/views/check-and-send.njk (hunk 2 above)
Comment (karthikkannankainos) [reply src/main/features/claim/views/check-and-send.njk]: sorted it.

**Pull request merged**